Resources/Locale/en-US/accessories/human-hair.ftl

```diff
@@ -124,6 +124,8 @@ marking-HumanHairKagami = Pigtails
 marking-HumanHairPigtails = Pigtails 2
 marking-HumanHairPigtails2 = Pigtails 3
 marking-HumanHairPixie = Pixie Cut
+# Floofstation - Polse
+marking-HumanHairPolseSpecial = Polse
```

Comment:
While this is not a wrong way to do it, it would be better to place this in the floofstation namespace (Resources/Locale/en-US/Floof/accessories/human-hair.ftl)